### ISSUE TITLE
left click new tab should just open the actual hierarchy

### ISSUE THREAD

**Issue by hoodriverheather** [Labcas-NIST/Labcas-ui#35]
Open new tab for both visualization as well as virtual hierarchy datasets

**Comment by yuliujpl** [comment]
Opens new tab now for both multiqc icon and virtual hierachy. However, if you click on multiple virtual hierarchies which opens multiple new tabs, if you refresh a tab that's older than the most recently generated tab, that older tab will refresh into the contents of the newer tab. This is because the virtual dataset webpage uses user session saved in your browser to generate the content, so it will use the most recently clicked virtual hierarchy dataset as a reference point to what content to generate. Hope that doesn't become confusing for the users!

**Comment by hoodriverheather** [comment]
@yuliujpl This works on my mac when i use command + click. A new tab opens with the correct page. If I control + click and then select open new tab, I'm not taken to the new page. I'm still on the page I control + clicked. Maybe this is OK?

**Comment by yuliujpl** [comment]
@hoodriverheather yup! Now, you don't need to do command or control click, just clicking will open a new page!

**Comment by hoodriverheather** [comment]
@yuliujpl I think putting it back to the way it was would be better. The command+click. Otherwise, I end up with lots of tabs open without expecting this behavior.

**Comment by hoodriverheather** [comment]
@yuliujpl Thanks for putting this back! Do you want to keep this issue open to eventually add the left click, open new tab as a feature?

**Comment by yuliujpl** [comment]
right click now opens a new tab, not able to get it to work where you get to select new tab though! However, the different outcomes from right/left click should hopefully be sufficient?
